vote/views.py

```diff
@@ -47,7 +47,7 @@ def constituency(request):
     return render(request,"vote/vote_by_constituency.html",context)
 
 # ##############################################################################################
-@login_required()
+@login_required(login_url='/login/')
 def polling_station(request):
```

Comment:
Login user's info shows in the address bar
When a user enters an unacceptable login, the system raise an error.
The data entered by the EC officer does not get to the blockchain. The blockchain is blank
User ( general public) needs no login to view votes but the system only allow login users to view some part of the vote.
![black_chain_emp](https://user-images.githubusercontent.com/47414553/149424908-5e9f9786-cb9d-4f73-b224-21af469f40ab.png)
![data_from_ec](https://user-images.githubusercontent.com/47414553/149424913-28df0fb0-0cea-44e4-963f-16f975f0c8c4.png)
![login_with_wrongdetails](https://user-images.githubusercontent.com/47414553/149424918-dc10f6c2-aa01-4d0b-b5be-cdafcbc6abb5.png)
![user_infor_inaddress_bar](https://user-images.githubusercontent.com/47414553/149424928-3a05749c-0053-47c0-9a30-b59231900be8.png)